mqtt_base_block.py

```diff
@@ -30,9 +30,8 @@ def stop(self):
 
     def connect(self):
```

Comment:
As well as `_disconnect` and really any method that isn't being used by another class that would call this block. Things like `configure`, `start`, `stop` and `process_signals` are called by the service and block router so that's why they don't have the underscore.